### PR TITLE
Add Social Profile

### PR DESCRIPTION
**Pull request by hoangdatngo**
This is the work for this issue: #63 

I have added links to various social profile of some of the speakers in the Fossasia OpenTechSummit Vietnam.

The deployment link: https://hoangdatngo.github.io/vn.opentechsummit.asia/

@mariobehling, @kushthedude, @hpdang Can any mentor please review my PR?

### CODE REVIEW

**Comment by hoangdatngo** [comment]
@liveHarshit, @kushthedude, @mariobehling Can any mentor please review this. I have been waiting for 2 days.

**Review by kushthedude — APPROVED** [review]
Looks good

**Comment by hoangdatngo** [comment]
@kushthedude Can you please merge my PR?

**Comment by kushthedude** [comment]
Actually I dont have merge rights for this repository.
@mariobehling @hpdang Please check and merge it.

**Comment by hoangdatngo** [comment]
@kushthedude ok, thanks. You review so fast!

File: index.html [hunk 1]
```diff
@@ -551,7 +555,8 @@ <h1>Speakers</h1>
                                 <a href="https://github.com/rbscholtus"><i class="icon fa fa-github"></i></a>
                                 <a href="https://twitter.com/rbscholtus"><i class="icon fa fa-twitter"></i></a>
                                 <a href="https://www.facebook.com/barend.scholtus"><i class="icon fa fa-facebook"></i></a>
-                  </div>
+				<a href="https://www.instagram.com/barend.scholtus/"><i class="icon fa fa-instagram"></i></a>
```
Comment (cweitat) [inline index.html:558]: follow the indent

File: index.html [hunk 2]
```diff
@@ -442,10 +442,13 @@ <h1>Speakers</h1>
                 <div class="hover-state text-center preserve3d">
                         <div class="social-links vertical-align">
                                 <a href="https://github.com/mariobehling/"><i class="icon fa fa-github"></i></a>
+				<a href="https://www.flickr.com/photos/rportrr/"><i class="icon fa fa-flickr"></i></a>
```
Comment (cweitat) [inline index.html:445]: what is this? how is it relevant?

File: index.html [hunk 3]
```diff
@@ -442,10 +442,13 @@ <h1>Speakers</h1>
...
                                 <a href="https://twitter.com/mariobehling" target="default"><i class="icon social_twitter"></i></a>
                                 <a href="https://www.facebook.com/mariobehling" target="default"><i class="icon social_facebook"></i></a>	
 				<a href="https://www.linkedin.com/in/mariobehling" target="_self"><i class="icon social_linkedin"></i></a>
-                            </div>
+				<a href="https://www.youtube.com/user/mariobehling" target="default"><i class="icon social_youtube"></i></a>
+				<a href="https://blog.ffii.org/author/mariobehling/" target="_self"><i class="icon fa fa-external-link"></i></a>
```
Comment (cweitat) [inline index.html:450]: how is link that is updated at 2015 useful?

File: index.html [hunk 4]
```diff
@@ -442,10 +442,13 @@ <h1>Speakers</h1>
                 <div class="hover-state text-center preserve3d">
                         <div class="social-links vertical-align">
                                 <a href="https://github.com/mariobehling/"><i class="icon fa fa-github"></i></a>
+				<a href="https://www.flickr.com/photos/rportrr/"><i class="icon fa fa-flickr"></i></a>
                                 <a href="https://twitter.com/mariobehling" target="default"><i class="icon social_twitter"></i></a>
                                 <a href="https://www.facebook.com/mariobehling" target="default"><i class="icon social_facebook"></i></a>	
 				<a href="https://www.linkedin.com/in/mariobehling" target="_self"><i class="icon social_linkedin"></i></a>
-                            </div>
+				<a href="https://www.youtube.com/user/mariobehling" target="default"><i class="icon social_youtube"></i></a>
```
Comment (cweitat) [inline index.html:449]: how is the channel useful in telling the profile of person??

File: index.html [hunk 5]
```diff
@@ -737,6 +742,9 @@ <h1>Speakers</h1>
               <div class="image-holder">
                 <img alt="Nguyen Tan Trieu" src="img/NguyenTanTrieu.jpg" width="300px">
                 <div class="hover-state text-center preserve3d">
+					<div class="social-links vertical-align">
+						<a href="https://www.linkedin.com/in/tantrieuf31/"><i class="icon fa fa-linkedin"></i></a>
+					</div>
```
Comment (cweitat) [inline index.html:747]: follow the indent

File: index.html [hunk 6]
```diff
@@ -750,6 +758,9 @@ <h1>Speakers</h1>
               <div class="image-holder">
                 <img alt="Trang Ta" src="img/trangta.jpeg" width="300px">
                 <div class="hover-state text-center preserve3d">
+					<div class="social-links vertical-align">
+						<a href="https://www.linkedin.com/in/trang1/"<i class="icon fa fa-linkedin"></i></a>
+					</div>
```
Comment (cweitat) [inline index.html:763]: follow the indent

File: index.html [hunk 7]
```diff
@@ -767,7 +778,8 @@ <h1>Speakers</h1>
                                 <a href="https://twitter.com/felixbecquart"><i class="icon social_twitter"></i></a>
                                 <a href="https://github.com/FelixBecquart1990"><i class="icon fa fa-github"></i></a>
                                 <a href="https://www.facebook.com/felix.becquart"><i class="icon social_facebook"></i></a>
-                            </div>
+				<a href="https://www.linkedin.com/in/f%C3%A9lix-becquart-8a5a46a6/"><i class="icon social_linkedin"></i></a>
```
Comment (cweitat) [inline index.html:781]: follow the indent

**Review by cweitat — CHANGES_REQUESTED** [review]

**Pull request closed without merging**